src/main/java/carpet_client/gui/ConfigListWidget.java

```diff
@@ -0,0 +1,102 @@
+package carpet_client.gui;
+
+import carpet_client.gui.entries.BooleanListEntry;
+import carpet_client.gui.entries.NumberListEntry;
+import carpet_client.gui.entries.StringListEntry;
+import carpet_client.utils.CarpetRules;
+import carpet_client.utils.ITooltipEntry;
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.gui.ParentElement;
+import net.minecraft.client.gui.widget.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Environment(EnvType.CLIENT)
```

Comment:
What are the benefits of using env annotations from fabric? That class will be added to the jar anyways, right?